scripts/package

```diff
@@ -1,23 +1,35 @@
-#!/bin/bash
+#!/bin/bash -x
 
-ARCH=${ARCH:-"amd64"}
-SUFFIX=""
-[ "${ARCH}" != "amd64" ] && SUFFIX="_${ARCH}"
+REPO=${REPO:-rancher}
+SCRIPTPATH="$(cd "$(dirname "$0")"; pwd -P)"
 
-source $(dirname $0)/version
+package() {
+  local os arch suffix tag repo env
+  os=$1
+  arch=$2
 
-cd $(dirname $0)/../package
+  [ "$os"   != "linux" ] && suffix="${suffix}_${os}"
+  [ "$arch" != "amd64" ] && suffix="${suffix}_${arch}"
 
-TAG=${TAG:-${VERSION}${SUFFIX}}
-REPO=${REPO:-rancher}
+  if $(echo ${tag} | grep -q dirty); then
+      tag=dev
+  fi
+  tag="${VERSION}${suffix}"
```

Comment:
What kind of tag is this going to produce? Examples please.

Reply:
include the image names too

Reply:
As @deniseschannon comments in https://github.com/rancher/rancher-catalog/pull/1026, I will use the same tag as linux version. I think that the image name will be like `rancher/metadata-windows:<tag>` if Denise agrees.